backend/src/resolvers.js

```diff
@@ -12,78 +14,96 @@ export default {
   },
 
   Mutation: {
-    createPost: (parent, args, context) => {
-      // throw error if user does not exist
-      const user = context.dataSources.db.getUser(args.userId);
-      if (!user) {
-        throw new UserInputError('Invalid user', {
-          invalidArgs: 'Userid does not exist',
+    login: (parent, args, context) => {
+      const { email, password } = args;
+
+      if (password.length < 8) {
```

Comment:
Due to missing type checking it would be possible to pass an array (which also has a `length` method) to the args which will result in a true statement.